SDL/BinSkimSetup.ps1

```diff
@@ -0,0 +1,38 @@
+﻿[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True)]
+    [string]$buildDirectory,
+    
+    [Parameter(Mandatory=$True)]
+    [string]$binSkimDirectory
+)
+
+#$buildDirectory = "C:\Repos\SDK\bin\Release\Src\"
+#$binSkimDirectory = "C:\Repos\SDK\binSkim\"
+
+Write-Host "`nPARAMETERS:";
+Write-Host "`tbuildDirectory:" $buildDirectory;
+Write-Host "`tbinSkimDirectory:" $binSkimDirectory;
+
+# don't need to clean folder on build server, but is needed for local dev
+if (Test-Path $binSkimDirectory) { Remove-Item $binSkimDirectory -Recurse; }
```

Comment:
Adding the following to your Test-Path will help out a bit in some cases:

    if (Test-Path -Path $binSkimDirectory -ErrorAction SilentlyContinue) { . . . }